core/drivers/crypto/crypto_api/include/drvcrypt_cipher.h

```diff
@@ -0,0 +1,73 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2018-2020 NXP
+ *
+ * Cipher interface calling the HW crypto driver.
+ */
+#ifndef __DRVCRYPT_CIPHER_H__
+#define __DRVCRYPT_CIPHER_H__
+
+#include <crypto/crypto_impl.h>
+#include <tee_api_types.h>
+
+/*
+ * Format the CIPHER context to keep the reference to the
+ * operation driver
```

Comment:
Suggest to simplify description into `Cipher operation context`

Reply:
Ok